main.py

```diff
@@ -131,5 +131,6 @@ def play_game(self):
             self.clean_up()
 
 
-game = Game()
-game.play_game()
+if __name__ == 'main':
```

Comment:
It looks like this branch was branched off your previous branch. Nothing technically wrong with that, but doing so by accident can cause all kinds of headaches. If you merge in #6 to master, check out to master, pull, checkout to this branch, then `git merge dev` this will go away. I'm going to hold off on approving this PR until this is done. Kind of annoying, but it will help reinforce the process.

Reply:
Thank you for pointing that out. I have experience with a teammate on a project doing exactly that same thing -- a headache indeed. I believe I fixed it.